test/ringpop-assert.js

```diff
@@ -896,7 +896,7 @@ function piggyback(tc, opts) {
 
     if(opts.subjectIx === 'sut') {
         update.address = tc.sutHostPort;
-        update.sourceIncarnationNumber = tc.test_state['sutIncarnationNumber'];
```

Comment:
It is hard to see without more context, but the `piggyback` function consists of roughly two parts. The first part is about the source of the gossip, the second part is about the subject. We are here in the part that is about the subject.

Even though we are setting the subject information this line had a bug where the incarnation number of the source of the gossip was set to the incarnation number of the sut. If you look at a bit more context below this line you will see that all branches set the `incarnationNumber` except for this branch.